nextroute/schema/output.go

```diff
@@ -2,13 +2,16 @@ package schema
 
 import (
 	"time"
+
+	"github.com/nextmv-io/sdk/nextroute"
 )
 
 // SolutionOutput represents a solutions as JSON.
 type SolutionOutput struct {
-	Unplanned []StopOutput    `json:"unplanned"`
-	Vehicles  []VehicleOutput `json:"vehicles"`
-	Objective ObjectiveOutput `json:"objective"`
+	Unplanned []StopOutput               `json:"unplanned"`
+	Vehicles  []VehicleOutput            `json:"vehicles"`
+	Objective ObjectiveOutput            `json:"objective"`
+	NextCheck *nextroute.NextCheckOutput `json:"next_check,omitempty"`
```

Comment:
This is a good as a place as any for this comment so here I go.

I think the API can be improved a bit. All exported symbols are named `NextCheck...`. This hints to the fact that it belongs in its own package. I would create a package under `nextroute` and define all the symbols there, minus the `NextCheck...` prefix of course.

Instead of `nextroute.NextCheckOutput` it then becomes `package.Output`, which is more aligned with Go design guides of placing stuff that belongs together in a package.

Regarding the package name, I'm not a huge fan of proprietary names. In this case, it is clear that the checker is for `nextroute`, so I don't see the necessity for adding yet another `next` word to it.

I would create the `nextroute/check` package, and thus the usage is:

- `check.Model`
- `check.Solution`
- `check.Options`
- `check.Output`

Notice in the suggestion that the type reads more smoothly.

```suggestion
	Check *check.Output `json:"check,omitempty"`
```

Reply:
Makes perfect sense, have created a package and moved stuff, changed verbosity option to string